### ISSUE TITLE
RequireAll::LoadError with latest master (94288599f9e1)

### ISSUE THREAD

**Issue by cesarizu** [cameronsutter/odyssey#13]
cameronsutter/odyssey#8 fixed the `require_all >= 2.0` problems but the latest master seem to have regressed

```
RequireAll::LoadError:
  no such file to load -- /home/travis/.rvm/gems/ruby-2.4.4/bundler/gems/odyssey-94288599f9e1/lib/formulas/formula
# /home/travis/.rvm/gems/ruby-2.4.4/gems/require_all-2.0.0/lib/require_all.rb:81:in `rescue in require_all'
# /home/travis/.rvm/gems/ruby-2.4.4/gems/require_all-2.0.0/lib/require_all.rb:58:in `require_all'
# /home/travis/.rvm/gems/ruby-2.4.4/gems/require_all-2.0.0/lib/require_all.rb:118:in `block in require_rel'
# /home/travis/.rvm/gems/ruby-2.4.4/gems/require_all-2.0.0/lib/require_all.rb:117:in `each'
# /home/travis/.rvm/gems/ruby-2.4.4/gems/require_all-2.0.0/lib/require_all.rb:117:in `require_rel'
# /home/travis/.rvm/gems/ruby-2.4.4/bundler/gems/odyssey-94288599f9e1/lib/odyssey.rb:63:in `<top (required)>'
# ./test/dummy/config/application.rb:18:in `<top (required)>'
# ./test/dummy/config/environment.rb:2:in `require_relative'
# ./test/dummy/config/environment.rb:2:in `<top (required)>'
# ./spec/rails_helper.rb:6:in `require'
# ./spec/rails_helper.rb:6:in `<top (required)>'
```

**Comment by jethrodaniel** [comment]
Comparing the git history from https://github.com/cameronsutter/odyssey/pull/8 to `94288599f9e1`, we can see why

```
λ git diff 97e4238e3..94288599f9e1
diff --git a/lib/formulas/formula.rb b/lib/formulas/_formula.rb
similarity index 100%
rename from lib/formulas/formula.rb
rename to lib/formulas/_formula.rb
```

This was fixed by `f60e623`, though, and released by the latest version bump (Wed Dec 19 17:42:38 2018, commit `54316ce7bfc068a`) to `0.3.0`.

~~In fact, I'm not sure why you're using such an old version of this gem.~~

You're probably not using `0.3.0` because it's not on Rubygems yet.

I've created an issue to push the latest version bump to Rubygems, which should fix this.
